tutorials/3d/gi_probes.rst

```diff
@@ -40,7 +40,7 @@ the geometry that will be affected.
 
 .. image:: img/giprobe_wrap.png
 
-Afterwards, make sure to enable the geometry will be baked. This is important in
+Afterwards, make sure to enable the "Use In Baked" property on the geometry instances. This is important in
 order for ``GIProbe`` to recognize objects, otherwise they will be ignored:
```

Comment:
```suggestion
Afterwards, make sure to enable the **Use In Baked Light** property on the geometry instances
in the inspector. This is required for ``GIProbe`` to recognize objects,
otherwise they will be ignored:
```